Find the common symbol.md

```diff
@@ -0,0 +1,47 @@
+# Summary
+In this experiment each subject is given a card on which there appears a set of five (out of six possible) symbols. Each subject's card is different from all the others in that the symbol lacking is a different symbol in each case. Thus, in any set of five cards there is only one symbol in common. The task is for every subject to find the common symbol.
+
+# References
+Main paper: Leavitt, H. J. (1951). Some effects of certain communication patterns on group performance. The Journal of Abnormal and Social Psychology, 46(1), 38–50. https://doi.org/10.1037/h0057189
+
+# Stimuli
+## The visual components
+A card with a set of five out of six symbols as follows are presented to the subjects:
+
+![Alt text](/images/Find_the_common_symbol.jpg)
+
```

Comment:
I would add the questionnaire questions listed in the results section: 
- How much did you enjoy your job? 
- Did your group have a leader, if so who? 
- Describe briefly the organization of your group.
- How did you like your job in the group? 
- See if you can recall how you felt about thejob as you went along. Draw the curve below. 
- Was there anything at any time that kept your group from performing its best? If so, what? 
- Do you thinly your group could improve its efficiency? If so, how?
- Rate your group on the scale below